records_mover/records/records_directory.py

```diff
@@ -121,6 +121,7 @@ def _filename_of_url(self, url: str) -> str:
         return path.split('/')[-1]
 
     def copy_to(self, new_loc: BaseDirectoryUrl) -> 'RecordsDirectory':
+        logger.info(f"Copying files from {self.loc} to {new_loc}...")
```

Comment:
This is a copy of a records directory, which can be larger - so indicating the status of exactly what we're copying is helpful here.